ims-examples/epub-sr-remote.html

```diff
@@ -2,7 +2,7 @@
 <html>
   <head>
     <meta charset='utf-8'>
-    <script src='../js/deps/require.js' data-main='../js/profile-ims-default' async class='remove'></script>
+    <script src='https://purl.imsglobal.org/spec/respec-ims-default.js' async class='remove'></script>
```

Comment:
Should this be a relatively sourced file?  If not, perhaps the purl location needs to be updated (if it hasn't already)?

Reply:
I made a few pairs of documents in ims-examples. Each pair has the same document contents, but used a different copy of ims profile. The "-local" document uses the local copy of Respec (localhost:5000 is required) and the "-remote" document uses the remote profile currently in production. This allowed me to load both and compare them for differences. I think it will be helpful to keep both as is for future development.

I think I missed the step of asking you to update https://purl.imsglobal.org/spec/respec-ims-default.js with the new version. It is only used while documents are being edited. Once they are published the documents use static data.

I'll look back at what I sent you before LILI. If respec-ims-default.js is not included, I'll send a copy.

Reply:
This branch in Spec-Central has the new IMS profile (respec-ims-default.js) which should be published to the purl server.

https://github.com/IMSGlobal/spec-central/tree/update_respec/js

Reply:
I merged the https://github.com/IMSGlobal/spec-central/tree/update_respec branch into master. So now https://github.com/IMSGlobal/spec-central/tree/master/js has the updated profile.

Reply:
> This allowed me to load both and compare them for differences. 
> I think it will be helpful to keep both as is for future development.

10-4. Makes sense.  If that isn't already documented (I didn't check), ya might consider it.

> I think I missed the step of asking you to update https://purl.imsglobal.org/spec/respec-ims-default.js 
> with the new version. It is only used while documents are being edited. 

Ya, that is why I was mentioning it.  @LMattson would have to help you with the purl server.  I don't have access to it right now.